### PR TITLE
Add MEV to glossary

### LINKED ISSUE #13431: Maximum Extractable Value (MEV)
### Checklist

- [X] This is a term not already found in the glossary (for similar terms, please consider the benefits of a new term vs updating an existing term)
- [X] This term/definition is not a product advertisement or contain other promotional content
- [X] This term/definition is directly relevant to Ethereum

### Term name

Maximum Extractable Value (MEV)

### Term definition

Maximal extractable value (MEV) refers to the maximum value that can be extracted from block production in excess of the standard block reward and gas fees by including, excluding, and changing the order of transactions in a block.

### Sources, if any (please do not submit copyrighted content without appropriate approval)

https://ethereum.org/en/developers/docs/mev/

### Ethereum.org links

https://ethereum.org/en/developers/docs/mev/

### Additional context

_No response_

### Would you like to work on this issue?

- [ ] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by JoshDavisLight**
## Description
Added MEV to glossary and updated definition in json file.

## Related Issue
- Closes #13431

### CODE REVIEW

File: src/intl/en/glossary.json [hunk 1]
```diff
@@ -238,7 +238,7 @@
   "message-call-term": "Message call",
   "message-call-definition": "The act of passing a <a href=\"/glossary/#message\">message</a> from one account to another. If the destination account is associated with <a href=\"/glossary/#evm\">EVM</a> code, then the VM will be started with the state of that object and the message acted upon.",
   "mev-term": "MEV",
-  "mev-definition": "A mechanism prioritizing certain actions on a blockchain for a fee, influencing outcomes and the order of transactions.",
+  "mev-definition": "Maximal extractable value (MEV) refers to the maximum value that can be extracted from block production in excess of the standard block reward and gas fees by including, excluding, and changing the order of transactions in a block.",
```
Comment (wackerow) [inline src/intl/en/glossary.json:241]: Would remove the term from its definition, and place in `mev-term` instead.

ie. `"mev-term": "Maximal extractable value (MEV)",`

```suggestion
  "mev-definition": "The maximum value that can be extracted from block production in excess of the standard block reward and gas fees by including, excluding, and changing the order of transactions in a block.",
```

**Comment by JoshDavisLight** [comment]
Thanks @wackerow. I took your suggestion, as well as added a link to the MEV page in the developer docs.

**Comment by Romansharar** [comment]
Hi Josh,

Thanks for implementing the suggestion! I appreciate your efforts in adding
the link to the MEV page in the developer docs as well. Great teamwork!

**Review by corwintines — APPROVED** [review]
LGTM, thanks @JoshDavisLight

**Comment by wackerow** [comment]
@all-contributors please add @JoshDavisLight for content

**Pull request merged**